src/_locales/de/messages.json

```diff
@@ -118,7 +199,7 @@
   },
 
   "tooltip_cFpiDomain": {
-    "message": "First-Party Isolation verhindert, dass Drittanbieter Cookies verwenden, um den Benutzer auf verschiedenen Websites zu identifizieren (siehe Optionen).",
```

Comment:
Could you review the German version of this translation which has been deleted please?